pkg/cache/postgres.go

```diff
@@ -51,6 +51,10 @@ func (c *postgresCache) AddCheckFromStatus(check pkg.Check, status pkg.CheckStat
 		return uuid.Nil, nil
 	}
 
+	if !check.Transformed {
```

Comment:
This flow is only for transformed checks but some untransformed checks were being processed as well, causing extra upserts